File: features/merchant/gold_merchant_common/src/main/java/com/tokopedia/gm/common/data/source/cloud/model/RequestCashbackModel.java
package com.tokopedia.gm.common.data.source.cloud.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by zulfikarrahman on 10/2/17.
 */

public class RequestCashbackModel {
    @SerializedName("product_id")
    private Long product_id;
    @SerializedName("cashback")
    private int cashback;

    public RequestCashbackModel(Long product_id, int cashback) {
        this.product_id = product_id;
        this.cashback = cashback;
    }
}

File: features/merchant/gold_merchant_common/src/main/java/com/tokopedia/gm/common/data/source/cloud/model/RequestCashbackModel.kt
package com.tokopedia.gm.common.data.source.cloud.model

import android.annotation.SuppressLint
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by zulfikarrahman on 10/2/17.
 */
data class RequestCashbackModel(
    @SuppressLint("Invalid Data Type")
    @SerializedName("product_id")
    @Expose
    private val product_id: Long,
    @SerializedName("cashback")
    @Expose
    private val cashback: Int
)